### ISSUE TITLE
Update X3ML Engine executor to support transformations using terminologies

### ISSUE THREAD

**Issue by ymark** [isl/x3ml#100]
X3MLEngine executor to use a file containing a skos taxonomy

**Comment by ymark** [comment]
X3ML Engine Executor has been updated to support tansformations using SKOS terminologies. 

More specifically a new method X3MLEngine.load( ) has been implemented that accepts (apart from the mappings), the terminology stream and the serialization format. The signature of the new method follows

```
X3MLEngine load(InputStream mappingsStream, InputStream terminologyStream, Lang terminologyLang)
```

The new functionality will be available from 1.8.2 onwards

**Comment by ymark** [comment]
Updated X3MLCommandLine to be able to execute X3ML Engine using terminologies from command line. More specifically the following parameters can be used:

```
-t,--terms <arg> the SKOS taxonomy
                            Option A-single file: -terms skosTerms.nt
                            Option B-URL: -terms @skos_terms_url

```
